testsuite/features/support/env.rb

```diff
@@ -23,8 +23,10 @@
 
 server = ENV['SERVER']
 $debug_mode = true if ENV['DEBUG']
-$long_tests_enabled = true if ENV['LONG_TESTS']
+$long_tests_enabled = !ENV['LONG_TESTS'].nil?
```

Comment:
In my humble opinion is less readable in that format :P

Reply:
You are absolutely right! I am going to use the previous one.